spec/index.html

```diff
@@ -1685,8 +1685,8 @@ <h2>Entailment rules (Informative)</h2>
       </tr>
       <tr >
         <td class="othertable"><dfn>GrdfD1</dfn></td>
-        <td class="othertable">   xxx aaa <code>"</code>sss<code>"^^</code>ddd <code>.</code> <br/>
-            for ddd in D</td>
+        <td class="othertable">  Any triple ttt such that "sss"^^ddd appears in ttt
+             for ddd in D </td> 
```

Comment:
Upon reviewing this and nearby tables, it appears that this markup *is* appropriate.
```suggestion
        <td class="othertable"> Any triple ttt such that <code>"</code>sss<code>"^^</code>ddd appears in ttt
            for ddd in D</td>
```